packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Icon, Dialog, Text } from '@deriv/components';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import { localize } from '@deriv/translations';
+
+type TGuideContent = {
+    dialog_options: { [key: string]: string };
+    faq_search_value: string;
+    is_dialog_open: boolean;
+    onOkButtonClick: () => void;
+    showVideoDialog: (type: string, component: HTMLVideoElement, url?: string) => void;
+};
+
+type TContentArray = {
+    id: number;
+    type: string;
+    content: string;
+    url?: string;
+};
+
+const contentArray: TContentArray[] = [
+    {
+        id: 1,
+        type: 'DBotVideo',
+        content: localize('DBot -- your automated trading partner'),
+        url: 'https://www.youtube.com/watch?v=tvBiEIq3G7k',
```

Comment:
After clicking on run I get this error:
Cross-Origin Read Blocking (CORB) blocked cross-origin respons

Reply:
done ✅